arp_poison/mitm.py

```diff
@@ -0,0 +1,45 @@
+from scapy.all import *
+from scapy.layers.l2 import Ether, ARP
+
+ARP_REQUEST = 1
+ARP_REPLY = 2
+ARP_REQUEST_TIMEOUT = 5  # seconds
+BROADCAST_MAC = "ff:ff:ff:ff:ff:ff"
+
+logging.getLogger(scapy.__name__).setLevel(logging.WARNING)
+logging.basicConfig(format='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p', level=logging.INFO)
+logger = logging.getLogger()
+
+
+def _get_mac(target_ip):
```

Comment:
add docstr to each function (parameters, description). pycharm should suggest the format automatically

Reply:
kk - thought _get_mac was self explanatory